src/forms/textInput.test.jsx

```diff
@@ -21,12 +32,14 @@ describe('TextInput', function() {
 			required: true,
 		};
 		textInputComponent = TestUtils.renderIntoDocument(
```

Comment:
since I won't be using that in these tests I'm reluctant to add it. I will when I do though!